calculator.c

```diff
@@ -0,0 +1,117 @@
+#include <stdio.h>
+#include <ctype.h>
+#include <stdlib.h>
+#define MAX_SIZE 100
+
+int operation(int a, int b, char op)
```

Comment:
apply code formatting and match function names with this standards, take care of this at all places.

Use snake_case for function names.
Start function names with a verb to indicate action.
Keep names concise but meaningful.
ex :
int calculate_sum(int a, int b);
void display_message(const char *message);